### PR TITLE
modified for LANGUAGE env for chrome in ubuntu

### PR DESCRIPTION
**Pull request by ypwalter**

### CODE REVIEW

**Comment by ypwalter** [comment]
https://github.com/Mozilla-TWQA/Hasal/issues/540

**Comment by askeing** [comment]
fix #540

**Comment by ypwalter** [comment]
@ShakoHo@askeing r?

File: lib/browser/chrome.py [hunk 1]
```diff
@@ -23,10 +22,8 @@ def get_browser_settings(self, **kwargs):
         self.launch_cmd = [self.command,
                            "--window-size=" + str(self.windows_size_width) + "," + str(self.window_size_height)]
 
-        if self.current_platform_name == "linux2":
-            self.launch_cmd = [ubuntu_language_prefix] + self.launch_cmd
-        elif self.current_platform_name.startswith('win'):
-            self.launch_cmd.extend([windows_language_postfix])
+        if self.current_platform_name.startswith('win'):
```
Comment (askeing) [inline lib/browser/chrome.py]: any reason using `.startswith('win')` here, not `== 'win32'`?

File: lib/browser/base.py [hunk 2]
```diff
@@ -18,6 +18,8 @@ def __init__(self, window_size_height, windows_size_width, **kwargs):
         self.windows_size_width = str(windows_size_width)
         self.get_browser_settings(**kwargs)
         self.test_env = os.environ.copy()
+        if self.current_platform_name == "linux2":
+            self.test_env['LANGUAGE'] = "en-US"
```
Comment (askeing) [inline lib/browser/base.py]: how about putting these two line in `BrowserChrome. get_browser_settings ()` method?

File: lib/browser/chrome.py (hunk 1 above)
Comment (ypwalter) [inline lib/browser/chrome.py]: I didn't know it's win32 for all windows platform including 64bit. Thanks.

**Comment by ypwalter** [comment]
@askeing  r?

**Comment by askeing** [comment]
lgtm r+

**Pull request merged**